backend/lucene/src/main/java/org/hibernate/search/backend/lucene/types/codec/impl/LuceneStringFieldCodec.java

```diff
@@ -110,7 +109,7 @@ public String normalize(String absoluteFieldPath, String value) {
 		}
 
 		return analyzerOrNormalizer != null
-				? AnalyzerUtils.normalize( analyzerOrNormalizer, absoluteFieldPath, value )
+				? analyzerOrNormalizer.normalize( absoluteFieldPath, value ).utf8ToString()
```

Comment:
I think most of the callers of `normalize()` convert the String you return to a `BytesRef`. So maybe it would be better to drop the extra `utf8ToString()` call and make this method return a `BytesRef` directly?